cmd/nerdctl/container_start_linux_test.go

```diff
@@ -61,3 +61,45 @@ func TestStartDetachKeys(t *testing.T) {
 	container = base.InspectContainer(containerName)
 	assert.Equal(base.T, container.State.Running, true)
 }
+
+func TestStartInteractive(t *testing.T) {
+	t.Parallel()
+
+	base := testutil.NewBase(t)
+	containerName := testutil.Identifier(t)
+
+	tearDown := func() {
+		base.Cmd("container", "rm", "-f", containerName).AssertOK()
+		base.Cmd("container", "rm", "-f", containerName+"-i").AssertOK()
+	}
+
+	tearDown()
+	t.Cleanup(tearDown)
+
+	needle := "going to be echo-ed from stdin"
+
+	base.Cmd("create", "--name", containerName, testutil.CommonImage).AssertOK()
+	base.Cmd("container", "start", containerName).
+		CmdOption(testutil.WithStdin(strings.NewReader("echo " + needle))).
+		AssertOutExactly(containerName + "\n")
+
+	cmd := base.Cmd("container", "start", "-i", containerName).
+		CmdOption(testutil.WithStdin(strings.NewReader("echo " + needle)))
+
+	// XXX docker behavior does not make much sense here
+	if base.Target == testutil.Docker {
+		cmd.AssertOutExactly("")
+	} else {
+		cmd.AssertOutExactly(needle + "\n")
+	}
```

Comment:
Can we just follow Docker's behavior, even if it does not make much sense?

Reply:
Of course. Let me look how to do that.